src/Pages/URLShortener/URLShortener.js

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useState } from 'react';
+
+import Header from '../../Components/Header/Header';
+import './URLShortener-page.css';
+import { getAllURLs, createURL, deleteURL } from '../../APIFunctions/URLShortener';
+import { Container, Button, Row, Col, Input } from 'reactstrap';
+
+export default function URLShortenerPage(props) {
+  const [URL, setURL] = useState();
+  const [alias, setAlias] = useState();
+  const [allURLs, setAllURLs] = useState([]);
+  const [deleteAlias, setDeleteAlias] = useState();
```

Comment:
add 2 more hooks here called `loading` and `error` which defaults to true and null respectively

in `getURLsFromDB` set loading to `false` after we get a response from the backend.

if there is an error, set the error hook to whatever the `URLsFromDB.error` is

then in the react code, only return the `<Container>` and children if `error` is null and `loading` is false